core/payment/src/dao/agreement.rs

```diff
@@ -115,7 +112,6 @@ pub fn set_amount_accepted(
     let agreement: ReadObj = dsl::pay_agreement
         .find((agreement_id, owner_id))
         .first(conn)?;
-    assert!(total_amount_accepted >= &agreement.total_amount_accepted); // TODO: Remove when payment service is production-ready.
```

Comment:
added check to protect from this: https://github.com/golemfactory/yagna/pull/1195/files#diff-41af9103ba1479fa3115da90a8b2647510a72d039b266afd4fea4c97a5bb587eR325